tools-local/targetgroups.props

```diff
@@ -168,6 +168,20 @@
       <NuGetTargetMoniker>.NETCoreApp,Version=v1.2</NuGetTargetMoniker>
       <NuGetTargetMonikerShort>netcoreapp1.2</NuGetTargetMonikerShort>
     </TargetGroups>
```

Comment:
You can delete the `corert` Group while you are on this - it was never used.

Reply:
Similar edits should be in pkg\Microsoft.NETCore.Platforms\runtime.json.

Reply:
Done